guides/v2.2/cloud/live/sens-data-over.md

```diff
@@ -140,70 +140,88 @@ The **Pro plan** environment high-level overview of this process:
 
 To add new configurations, modify your environment through the Magento Admin panel and run the command again to generate the file. Any new configurations are appended to the code in the file. Push it to Git to update.
 
 <div class="bs-callout bs-callout-warning" markdown="1">
-While you can manually edit `config.php` in Staging and Production, we don't recommend it. The file helps keep all of your configurations consistent across all of your environments.
+While you can manually edit the `config.php` file in the Staging and Production environments, we do not recommend it. The file helps to keep all configurations consistent across all environments.
 
-You should never delete `config.php` to rebuild it. Deleting the file can remove specific configurations and settings required for build and deploy processes.
+Never delete the `config.php` file to rebuild it. Deleting the file can remove specific configurations and settings required for build and deploy processes.
 </div>
 
-## Migrate config.local.php to config.php {#migrate}
-If you upgrade to {{site.data.var.ece}} 2.2 or later, you may want to migrate settings from `config.local.php` to your new `config.php` file. If the configuration settings in your Magento Admin match the contents of the file, you can follow the instructions to generate and add `config.php`.
+## Migrate configurations
+If you upgrade to {{site.data.var.ece}} 2.2 or later, you may want to migrate settings from the `config.local.php` file to your new `config.php` file. If the configuration settings in your Magento Admin panel match the contents of the file, follow the instructions to generate and add the `config.php` file.
 
-If they differ, you can append content from `config.local.php` to your new `config.php` file:
+If they differ, you can append content from the `config.local.php` file to your new `config.php` file:
 
 1. Follow instructions to generate the `config.php` file using the [recommended method](#cloud-config-specific-recomm).
-2. Open `config.php`and delete the last line.
-3. Open `config.local.php`and copy the contents.
-4. Paste the contents into `config.php`, save, and complete adding it to Git.
+2. Open the `config.php` file and delete the last line.
+3. Open the `config.local.php` file and copy the contents.
+4. Paste the contents into the `config.php` file, save, and complete adding it to Git.
 5. Deploy across your environments.
 
-You only need to complete this migration once. When you need to update the file, you will always update the new `config.php`.
+You only need to complete this migration once. When you need to update the file, always update the `config.php` file.
+
+## Change locales
+You can change your store locales without following a complex configuration import and export process, _if_ you have [SCD_ON_DEMAND]({{ page.baseurl }}/cloud/env/variables-intro.html#scd_on_demand) enabled for your Staging and Production environments. You can update the locales using the Admin panel.
```

Comment:
I would not mention environment types here, if SCD_ON_DEMAND is enabled (on any env) you can update locales using Admin panel.

Reply:
Okay!, I will correct this in the release branch. Thanks Nadiia!